magma/lib/magma/validation/attribute.rb

```diff
@@ -6,7 +6,7 @@ def initialize(model, validator)
         @attribute_validations = {}
       end
 
-      def validate(document)
+      def validate(record_name, document)
```

Comment:
This also seems to contradict the above order.  Is the inner model validation's param ordering document, record_name or the other way around?

Maybe consider switching these to named args: `def validate(record_name:, document:)`